src/jquery.floatThead.js

```diff
@@ -95,6 +95,10 @@
         return result;
       };
     };
+    that.isFunction = function(obj) {
```

Comment:
Agghh, it does work. I am not sure what happened. The fix is just removing the window._

I removed the isFunction part and updated. Sorry for the confusion.